test/cmdlineTests/yul_eof_container/output

```diff
@@ -0,0 +1,5 @@
+
+======= yul_eof_container/input.yul (EVM) =======
+
+Binary representation:
+ef00010100040200010003030004001a001b003b001b040000000080ffff5f80fdef00010100040200010007040000000080ffff5f805260205ffdef00010100040200010008040000000080ffff60015f5260205ffdef00010100040200010008030001001b040000000080ffff60025f5260205ffdef00010100040200010008040000000080ffff60205f5260205ffdef00010100040200010008040000000080ffff60035f5260205ffd
```

Comment:
The plan is to use default outputs for `--strict-assembly` input but assembly output is not supported for EOF yet. When it will be I will remove `--bin` and we will have `Text representation:` output also. 

For `--opcodes` now unfortunately I get this error `Error: The following outputs are not supported in assembler mode: --opcodes. `

Reply:
Text representation sounds good, as long as this is a format we're going to commit to and is suitable to show to users.

If not, we can still use it internally though - we can create a new test case type based on `YulOptimizerTest` and use it there just for testing.